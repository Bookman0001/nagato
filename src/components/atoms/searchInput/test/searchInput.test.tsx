import { shallow } from 'enzyme'

import SearchInput from '../'

test('SearchInput', () => {
  const component = shallow(
    <SearchInput
      placeholder={'test'}
      onChange={(e) => console.log(e.target.value)}
    />
  )
  expect(component).toMatchSnapshot()
})
