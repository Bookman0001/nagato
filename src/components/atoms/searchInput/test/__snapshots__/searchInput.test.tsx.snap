// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchInput 1`] = `
<styled.input
  onChange={[Function]}
  placeholder="test"
  type="search"
/>
`;
